sea-orm-cli/src/commands.rs

```diff
@@ -356,104 +359,123 @@ where
 
 #[cfg(test)]
 mod tests {
+    use clap::StructOpt;
+
     use super::*;
-    use crate::cli;
-    use clap::AppSettings;
+    use crate::{Cli, Commands};
 
     #[test]
     #[should_panic(
         expected = "called `Result::unwrap()` on an `Err` value: RelativeUrlWithoutBase"
     )]
     fn test_generate_entity_no_protocol() {
-        let matches = cli::build_cli()
-            .setting(AppSettings::NoBinaryName)
-            .get_matches_from(vec![
-                "generate",
-                "entity",
-                "--database-url",
-                "://root:root@localhost:3306/database",
-            ]);
-
-        smol::block_on(run_generate_command(matches.subcommand().1.unwrap())).unwrap();
+        let cli = Cli::parse_from(vec![
+            "generate",
+            "entity",
+            "--database-url",
+            "://root:root@localhost:3306/database",
+        ]);
```

Comment:
Hey @smonv, thanks for the contributions! I think we need to include the name of executable?

```suggestion
        let cli = Cli::parse_from(vec![
            "sea-orm-cli",
            "generate",
            "entity",
            "--database-url",
            "://root:root@localhost:3306/database",
        ]);
```

Reply:
thank @billy1624 , it working now